app/lists.rb

```diff
@@ -0,0 +1,43 @@
+module Listing
+  extend Discordrb::Commands::CommandContainer
+  
+  command :WingInitilise, description: "Initilise the array. In case of derps." do |event|
+	wing_list = Array.new
+  end
+
+  command :WingList, description: "Lists those who are requesting wing." do |event| #Simple, return the text file's contents. May consider making this a array. Also, admin lock this.
+    
+  end
+  
+  command :WingRemove, description: "Remove a person from the wing queue", min_args: 1, max_args: 1 do |event| #Remove someone who is on the wing list. Defo admin lock.
+	
+  end
+
+  command :WingMe, description: "Need a wing? &WingMe" #Adds the person who used the command to the wing list.
+	to_wing = user.name # The user who typed &WingMe
+	wing_list.push(to_wing) #add them to the array-list
```

Comment:
this is basically defining wing_list, and will quite possibly fail. Variables are method-scoped by default. If you define the variable in the module, it'll be useable. Modules are also weird in ruby...